shared/domain/src/main/java/gov/va/vro/model/AbdMedication.java

```diff
@@ -11,30 +12,37 @@
 
 @Getter
 @Setter
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class AbdMedication implements Comparable<AbdMedication> {
+  @EqualsAndHashCode.Include
   @Schema(description = "Status of medication", example = "Active")
   private String status;
 
   @Schema(description = "Notes of the medication order")
   private List<String> notes;
 
+  @EqualsAndHashCode.Include
   @Schema(description = "Medication name", example = "Hydrochlorothiazide 6.25 MG")
   private String description;
 
+  @EqualsAndHashCode.Include
   @Schema(description = "Number of refills remaining", example = "2")
   private int refills;
 
+  @EqualsAndHashCode.Include
   @Schema(description = "Medication can be for asthma", example = "true")
   private Boolean asthmaRelevant;
 
   private Boolean conditionRelated;
   private List<String> suggestedCategory;
 
+  @EqualsAndHashCode.Include
   @Schema(description = "Duration of the medication")
   private String duration;
 
+  @EqualsAndHashCode.Include
```

Comment:
Is it likely that the timestaps will match exactly? What is everything else is the same except the timestamp